### ISSUE TITLE
CVE-2016-1238 mitigation is ineffective

### ISSUE THREAD

**Issue by haarg** [Perl-Toolchain-Gang/File-Fetch#17]
The mitigation for CVE-2016-1238 added to this module is https://github.com/Perl-Toolchain-Gang/File-Fetch/blob/master/lib/File/Fetch.pm#L55-L56 :
```
### Fix CVE-2016-1238 ###
local $Module::Load::Conditional::FORCE_SAFE_INC = 1;
```
The problem is that this is at the top level of the module. It will only be in effect when loading the module itself, not when any of its methods are run. The variable instead needs to be set when `can_load` is run, similar to what #11 does.

**Comment by haarg** [comment]
This was fixed in 61805356d9a19db9290c87b92e6bce7878929acc, released with 1.00.

**Comment by karenetheridge** [comment]
...which was 2020-11-11, and first included with perl 5.33.4.
